flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/plan/nodes/exec/batch/CalcCompiledBatchTest.java

```diff
@@ -19,16 +19,16 @@
 package org.apache.flink.table.planner.plan.nodes.exec.batch;
 
 import org.apache.flink.table.planner.plan.nodes.exec.common.CalcTestPrograms;
-import org.apache.flink.table.planner.plan.nodes.exec.testutils.BatchCompiledPlanTestBase;
+import org.apache.flink.table.planner.plan.nodes.exec.testutils.CompiledBatchTestBase;
 import org.apache.flink.table.test.program.TableTestProgram;
 
 import java.util.Arrays;
 import java.util.List;
 
 /** Restore tests for {@link BatchExecCalc}. */
-public class CalcBatchCompiledPlanTest extends BatchCompiledPlanTestBase {
+public class CalcCompiledBatchTest extends CompiledBatchTestBase {
```

Comment:
I would still call it `BatchRestoreTest` it will make discussions easier and it also performs kind of a restore but from plan only.

Reply:
Are you looking for `CalcCompiledBatchTest` to change to `CalcBatchRestoreTest`?

Or are you suggesting that `CompiledBatchTestBase ` should change to `BatchRestoreTestBase `?

Lemme know which of those makes sense and I'll make the change.

Reply:
Sorry for the confusion. I thought we agreed on `BatchRestoreTestBase` and thus `CalcBatchRestoreTest`.

Reply:
Similar architecture than the streaming ones.

Reply:
Ok, I think I got the names updated as suggested.